client/src/components/AuditBoard/BallotAudit.tsx

```diff
@@ -193,68 +212,84 @@ const BallotAudit: React.FC<IProps> = ({
 }
 
-export const INVALID_WRITE_IN = 'INVALID_WRITE_IN'
-
 const BallotAuditContest = ({
   contest,
   interpretation,
   setInterpretation,
 }: IBallotAuditContestProps) => {
+  const {
+    choiceIds,
+    comment,
+    isBlankVoteChecked,
+    isContestNotOnBallotChecked,
+    isInvalidWriteInChecked,
+  } = interpretation
+
   const onCheckboxClick = (value: string) => (
     e: React.ChangeEvent<HTMLInputElement>
   ) => {
     const { checked } = e.currentTarget
-    if (
-      value === Interpretation.BLANK ||
-      value === Interpretation.CANT_AGREE ||
-      value === Interpretation.CONTEST_NOT_ON_BALLOT
-    ) {
-      setInterpretation({
-        ...interpretation,
-        interpretation: checked ? value : null,
-        choiceIds: [],
-        hasInvalidWriteIn: false,
-      })
-    } else if (value === INVALID_WRITE_IN) {
-      let newInterpretation: Interpretation | null
-      if (interpretation.choiceIds.length > 0) {
-        newInterpretation = Interpretation.VOTE
-      } else if (checked) {
-        newInterpretation = Interpretation.BLANK
+    if (value === Interpretation.BLANK) {
+      if (checked) {
+        setInterpretation({
+          ...interpretation,
+          choiceIds: [],
+          isBlankVoteChecked: true,
+          isContestNotOnBallotChecked: false,
+          isInvalidWriteInChecked: false,
+        })
       } else {
-        newInterpretation = null
+        setInterpretation({
+          ...interpretation,
+          isBlankVoteChecked: false,
+        })
       }
-      setInterpretation({
-        ...interpretation,
-        interpretation: newInterpretation,
-        hasInvalidWriteIn: checked,
-      })
-    } else {
-      const choiceIds = checked
-        ? [...interpretation.choiceIds, value]
-        : interpretation.choiceIds.filter(v => v !== value)
-      let newInterpretation: Interpretation | null
-      if (choiceIds.length > 0) {
-        newInterpretation = Interpretation.VOTE
-      } else if (interpretation.hasInvalidWriteIn) {
-        newInterpretation = Interpretation.BLANK
+    } else if (value === Interpretation.CONTEST_NOT_ON_BALLOT) {
+      if (checked) {
+        setInterpretation({
+          ...interpretation,
+          choiceIds: [],
+          isBlankVoteChecked: false,
+          isContestNotOnBallotChecked: true,
+          isInvalidWriteInChecked: false,
+        })
+      } else {
+        setInterpretation({
+          ...interpretation,
+          isContestNotOnBallotChecked: false,
+        })
+      }
+    } else if (value === INVALID_WRITE_IN) {
+      if (checked) {
+        setInterpretation({
+          ...interpretation,
+          isBlankVoteChecked: false,
+          isContestNotOnBallotChecked: false,
+          isInvalidWriteInChecked: true,
+        })
       } else {
-        newInterpretation = null
+        setInterpretation({
+          ...interpretation,
+          isInvalidWriteInChecked: false,
+        })
```

Comment:
In a vote-for-1, shouldn't selecting `Invalid Write-In` also clear the selected choices?

Reply:
maybe:

```ts
setInterpretation({
   ...interpretation,
   choices: contest.votesAllowed === 1 ? [] : interpretation.choices,
   isContestNotOnBallotChecked: false,
   isBlankVoteChecked: false,
   isInvalidWriteInChecked: checked,
})
```

Reply:
Ah actually @jonahkagan I'm now remembering why I didn't add logic like this. Even for a vote-for-1 today, we allow the audit board to select multiple choices since a voter may have overvoted. Likewise, I figured that a voter could specify both a choice and an invalid write-in, even for a vote-for-1. And the audit board UI should allow audit board members to transcribe exactly what they see on a ballot. Let me know if I'm missing something!